openmc/mgxs/mgxs.py

```diff
@@ -919,23 +919,6 @@ def load_from_statepoint(self, statepoint):
                   'linked with a summary file'
             raise ValueError(msg)
 
-        # Override the domain object that loaded from an OpenMC summary file
-        # NOTE: This is necessary for micro cross-sections which require
-        # the isotopic number densities as computed by OpenMC
```

Comment:
I went ahead and implemented this concept in "summary.py" and "mgxs/mgxs.py" - if you dislike it, I can trash that commit and try another approach.